share/spack/win-compiler-wrapper/cl.cxx

```diff
@@ -0,0 +1,440 @@
+/**
+ * @file cl.cxx
+ * @author John Parent (john.parent@kitware.com)
+ * @brief A C++ wrapper header file for the MSVC c and c++ compilers and linkers
```

Comment:
This part is copied from `cl.hxx` and could use minor revising (to be clear I see that not the entire comment is copied, but this paragraph is).